tests/profiler/alice_user_workflow_simplified_profiler_config.yml

```diff
@@ -1,38 +1,37 @@
+# Note: This config is simplified by assuming class_name, module_name, metric_domain_kwargs, column, etc where appropriate
+# Note: This simplification scheme has not been implemented, so please refer to the alice_user_workflow_verbose_profiler_config.yml for a working config
+# Simplification methods for ExpectationConfigurationBuilders are still being discussed
 variables:
   max_user_id: 999999999999
   min_timestamp: 2004-10-19 10:23:54
 rules:
-  # my_rule_for_user_ids:
-  #   domain_builder:
-  #     class_name: MyCustomSemanticTypeColumnDomainBuilder
-  #     module_name: plugins.profiler.domain_builder.my_custom_semantic_type_column_domain_builder
-  #     semantic_types:
-  #       - user_id
-  #   parameter_builders:
-  #     - name: my_min_user_id
-  #       class_name: MetricParameterBuilder
-  #       metric_name: column.min
-  #   expectation_configuration_builders:
-  #     - expectation_type: expect_column_values_to_be_between
-  #       min_value: $my_min_user_id.parameter.min_value
-  #       max_value: $variables.max_user_id
-  #     - expectation_type: expect_column_values_to_not_be_null
-  #     - expectation_type: expect_column_values_to_be_of_type
-  #       type_: INTEGER
+  my_rule_for_user_ids:
+    domain_builder:
+      class_name: MyCustomSemanticTypeColumnDomainBuilder
+      module_name: tests.profiler.plugins
+      semantic_types:
+       - user_id
+    parameter_builders:
+      - parameter_name: my_min_user_id # User defined
+        class_name: MetricParameterBuilder
+        metric_name: column.min
+    expectation_configuration_builders:
+      # Note: Simplification methods for ExpectationConfigurationBuilders are still being discussed
+      - expectation_type: expect_column_values_to_be_between
+        min_value: $parameter.my_min_user_id.column.min
+        max_value: $variables.max_user_id
+      - expectation_type: expect_column_values_to_not_be_null
+      - expectation_type: expect_column_values_to_be_of_type
+        type_: INTEGER
   my_rule_for_timestamps:
     domain_builder:
       class_name: SimpleColumnSuffixDomainBuilder
       column_name_suffixes:
         - _ts
-      user_input_list_of_domain_names:
-        - event_ts
-        - server_ts
-        - device_ts
```

Comment:
Did you mention that you might want to keep this in to demonstrate the functionality?